core-feature-pack/src/main/resources/content/domain/configuration/host-master.xml

```diff
@@ -5,6 +5,9 @@
    and does not itself directly control any servers.
 -->
 <host name="master" xmlns="urn:jboss:domain:3.0">
+    <extensions>
+        <extension module="org.jboss.as.jmx"/>
```

Comment:
If that happens the tests/config will have to move. We already use it although in other places, e.g. standalone.xml and host.xml